projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-js.component.ts

```diff
@@ -77,8 +78,14 @@ export class AngularPlugin extends BaseTelemetryPlugin {
                     _angularCfg = ctx.getExtCfg<IAngularExtensionConfig>(_self.identifier, defaultAngularExtensionConfig);
                     _propertiesPlugin = core.getPlugin<PropertiesPlugin>(PropertiesPluginIdentifier)?.plugin as PropertiesPlugin;
                     _analyticsPlugin = core.getPlugin<AnalyticsPlugin>(AnalyticsPluginIdentifier)?.plugin as AnalyticsPlugin;
+                    
+                    if (_angularCfg.enableInjector && _injector){
```

Comment:
Thought...

As we are passing the injector in do we either
- have this as "disableInjector" (instead of disableInjector)
- Or do we just expect that if we call `this._injector.get(ApplicationinsightsAngularpluginErrorService);` and it returns nothing then we fall back to the existing (which we are doing anyway)

Or if they are using an injector is it always passed?

Reply:
As injector is rarely used by user and maybe easy to use incorretly, I guess using enalbeInjector could help customer make sure that they are trying to use this special parameter?